src/modules/dom.js

```diff
@@ -72,6 +73,11 @@ const updateDOM = () => {
         });
       });
 
+      // Count number of games
+      const navGame = document.querySelector('.nav-games');
+      const numberOfDisplayedGames = countListOfGames(cards);
+      navGame.textContent = `Games (${numberOfDisplayedGames})`;
+
```

Comment:
[OPTIONAL] Good job with the lines of code, however, I think it's better to have it as a function or separate module so it can be reusable to determine child elements for any parent. This will ensure your code is not DRY